doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -48,18 +154,159 @@ to keep your client decoupled from implementation details.
+      if the value is not in the range (range includes the starting and ending values). I can be only used with
+      attributed of numeric or date types. E.g. <code>temperature!=10..20</code>matchs if the temperature is lower
+      than 10 or greater than 20 (it will not match if the tempeature is 10 or the temperature is 20).</li>
+    </ul>
+  </li>
+  <li>Greater than: <code>&gt;</code>. The value is a single element, e.g. <code>temperature&gt;40</code>. It
+  matches entities which attribute value is strictly greater than that value. I can be only used with attributes
+  of numeric or date types.</li>
+  <li>Lesser than: <code>&lt;</code>. The value is a single element, e.g. <code>temperature&lt;40</code>. It
+  matches entities which attribute value is strictly lesser than that value. I can be only used with attributes
+  of numeric or date types.</li>
+  <li>Greater or equal than: <code>&gt;=</code>. The value is a single element, e.g. <code>temperature&gt;=40</code>.
+  It matches entities which attribute value is greater than or equal to that value. I can be only used with attributed
+  of numeric or date types.</li>
+  <li>Lesser or equal than: <code>&lt;=</code>. The value is a single element, e.g. <code>temperature&lt;=40</code>. It
+  matches entities which attribute value is lesser than or equal to that value. I can be only used with attributed
+  of numeric or date types.</li>
+</ul>
+
+<p>In the case of equal or inequal, if the value to match include a <code>,</code>, you can use simple quote
+(<code>'</code>), e.g: <code>color=='light,green','deep,blue'</code>.</p>
+
+<p>Unary statements are composed of a prefix (either <code>+</code>or <code>-</code>) and are used in two situations:</p>
+
+<ul>
+  <li>To mean existence or no-existence of attributes. E.g. <code>+temperature</code>matches entities which have
+  a temperature attribute (no matter its value), while <code>-temperature</code>matches entities which don'e have
+  a temperature attribute.</li>
+  <li>To mean existence or no-existence of entity type, with the <code>type</code>keyword. E.g, <code>-type</code>
+  matches entities which don't have a type.</li>
+</ul>
+
+</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">geometry</td><td>
+
+<p>Defines a geografical area so only the entities located in that area matches the query. It is composed of list of
+tokens separated by <code>;</code>. The first token is the shape of the geometry, the rest of the tokens (if any) depends
+of the shape. In particular, the following geometries can be defined so far:</p>
+
+<ul>
+  <li><code>geometry=circle;radius:4000</code>, inside a circle (radius is in meters).</li>
+  <li><code>geometry=circle;radius:4000;external</code>, outside a circle (radius is in meters).</li>
+  <li><code>geometry=polygon</code>, inside a polygon.</li>
+  <li><code>geometry=polygon;external</code>, outside a polygon.</li>
+</ul>
+
```

Comment:
It could be interesting. Let's discuss details out of band.

NTC (in this PR).